content/en/blog/2023/sig-instrumentation-spotlight.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: blog
+title: "Spotlight on SIG Instrumentation"
+slug: sig-instrumentation-spotlight
+date: 2023-01-31
+canonicalUrl: https://www.kubernetes.dev/blog/2023/01/31/sig-instrumentation-spotlight-2023/
```

Comment:
Don't set a `canonicalUrl` for **this** PR. However, the article on https://blog.k8s.io/ does need that front matter variable set.